ci_framework/hooks/playbooks/fetch_compute_facts.yml

```diff
@@ -30,6 +30,8 @@
   connection: local
   tasks:
     - name: Load parameters
+      delegate_to: crc
```

Comment:
sooo we'd then take only those from the CRC host? Expecting CRC to get all the data for all the existing nodes in the deploy?